api/candidates/tests/test_views.py

```diff
@@ -26,7 +26,7 @@ def candidate():
 
 def test_get_candidate_by_cpf(client, candidate):
     cpf = candidate.cpf
-    response = client.get('/candidate/{}'.format(cpf), follow=True)
+    response = client.get('/candidate/cpf/{}'.format(cpf), follow=True)
```

Comment:
Django has its nice [util](https://docs.djangoproject.com/en/2.0/ref/urlresolvers/) which is `reverse()` that could help you out when you try to get the `URL` of some endpoint. So, if you change the URL, you do not have to change it on the test.